src/watchdog.rs

```diff
@@ -16,6 +16,14 @@
 //! The IWDG runs on a separate 40kHz low-accuracy clock (30kHz-60kHz). You may
 //! want to some buffer in your interval.
 //!
+//! Per default the iwdg continues to run even when you stopped execution of code via a debugger.
+//! You may want to disable the watchdog when the cpu is stopped
+//!
+//! ```
+//! let dbgmcu = p.DBGMCU;
+//! dbgmcu.apb1_fz.modify(|_, w| w.dbg_iwdg_stop().set_bit());
+//! ````
```

Comment:
Wrong use of backticks, also this might be interpreted as a standalone (even runnable) doctest which is probably not what we want.